framework/src/st_things/things_stack/cloud/cloud_manager.c

```diff
@@ -1580,6 +1582,11 @@ static char *make_cloud_address(char *ip, char *port, const char *ci_addr)
 		return NULL;
 	}
 
+	if (is_ci_addr_null == true) {
```

Comment:
Minor: Can we change it to **if(is_ci_addr_null)** ?

Reply:
@sushant-23 Could you please leave your opinion.
Our policy is that to reply every comment whether it is fix or not.
If you don't want to change then please leave why.